src/main/java/seedu/mtracker/console/EditInstrumentParser.java

```diff
@@ -116,13 +117,20 @@ public static void editExpiryParameter(HashSet<String> parametersGiven) {
         }
         TextUi.displayEditExpiry();
         String inputExpiry = getUserInput(WORKSPACE);
-        if (!Validate.isValidExpiry(inputExpiry)) {
+        while (!Validate.isValidExpiry(inputExpiry)) {
             ErrorMessage.displayEditExpiryError();
-            return;
+            inputExpiry = getUserInput(WORKSPACE);
         }
         editedParameters.put(EXPIRY_ATTRIBUTE, inputExpiry);
     }
 
+    public static void editDoneStatus(HashSet<String> parametersGiven) {
+        if (!parametersGiven.contains(DONE_ATTRIBUTE)) {
+            return;
+        }
+        editedParameters.put(DONE_ATTRIBUTE,DONE_ATTRIBUTE);
```

Comment:
```suggestion
        editedParameters.put(DONE_ATTRIBUTE, DONE_ATTRIBUTE);
```

Can get the user input like put in yes/no to set it as done or undone. The editing of status can go both ways